qa/Vagrantfile

```diff
@@ -0,0 +1,27 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+require_relative "./platform_config.rb"
+
+Vagrant.configure(2) do |config|
+  platforms = PlatformConfig.new
```

Comment:
I know you love this way of doing vagrant files, but can we keep the specific config areas for each platform? as soon as we've specific needs for a platform, or even a version, this kind of config file will be a mess.